src/containers/Tenant/Diagnostics/Overview/Overview.tsx

```diff
@@ -78,14 +80,26 @@ function Overview(props: OverviewProps) {
 
     const loading = schemaLoading || olapStatsLoading;
 
-    useAutofetcher(
-        (isBackground) => {
+    const childrenPaths = useTypedSelector(
+        (state) => selectSchemaChildrenPaths(state, currentSchemaPath),
+        shallowEqual,
```

Comment:
no need for shallow equality, we use reselect
if unwanted rerenders occur, this means there is an issue with how selectors are written

Reply:
This comment is for the future reference.

discussed this on a call; selectors are fine, the issue is unconditionally updating a field in the store, which causes reference update, so reselect can't help. This is an issue because of a circular reference: data fetch depends on children paths, and children paths are updated by this data fetch

shallow equality fixes this, so it is ok to keep it
but a better way would be to either get rid of the circular reference, or at least to move the shallow equality comparator from the component to the selector declaration. There were some issues with the latter previously though. So again, it is ok to keep it as is, if nothing else works.